index.html

```diff
@@ -0,0 +1,319 @@
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project">
+                        <p class="see-p">See Project</p>
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="card-works4">
+            <div class="snap4"></div>
+            <div class="left-block">
+                <div class="pt">
+                    <h2>Multi-Post</br>
+                        Stories</h2>
+                    <ul class="frame2-l">
+                        <li class="client">
+                            <h5 class="canopy">CANOPY</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="role">
+                            <h5 class="Back">Back End Dev</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="year-label">
+                            <h5 class="year">2015</h5>
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project-l">
+                        <p class="see-p">See Project</p>
```

Comment:
- [Optional] Please replace the  `p` tag and make use of a `span` tag, which is a generic inline container that can be used for styling purposes or to group text please I suggest you do this for all your buttons. Here's an example of how you could modify the code using a `span` tag:

```
<button class="see-project">
    <span class="see-span">See Project</span>
</button>
```